synfig-studio/po/POTFILES.in

```diff
@@ -410,6 +412,8 @@ src/synfigapp/actions/layerparamunsetstatic.cpp
 src/synfigapp/actions/layerparamunsetstatic.h
 src/synfigapp/actions/layerraise.cpp
 src/synfigapp/actions/layerraise.h
+src/synfigapp/actions/layerraisemax.cpp
+src/synfigapp/actions/layerraisemax.h
```

Comment:
maybe name file and action to "layer to top"?

Reply:
Done